tools/ci-build/runtime-versioner/Cargo.toml

```diff
@@ -18,6 +18,8 @@ anyhow = "1.0.75"
 camino = "1.1.6"
 clap = { version = "4.4.11", features = ["derive"] }
 crates-index = "2.3.0"
+fs_extra = "1.3.0"
```

Comment:
is `fs_extra` actually used? I removed it and everything still built / tested fine.